Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/EntityPropertyEditor.cpp

```diff
@@ -5922,6 +5929,13 @@ namespace AzToolsFramework
                 componentEditor->LeftComponentMode(componentModeTypes);
             }
 
+            // return to the scroll offset before Component Mode was activated
+            if (m_componentModeVerticalScrollOffset.has_value())
+            {
+                m_gui->m_componentList->verticalScrollBar()->setValue(m_componentModeVerticalScrollOffset.value());
+                m_componentModeVerticalScrollOffset = AZStd::nullopt;
+            }
+
```

Comment:
Hm. After looking at this a bit more, should we actually be doing this? It definitely makes sense to scroll _to_ the selected component when entering Component Mode, but it seems like it would be a bit jarring to scroll _away_ from it when exiting Component Mode, especially if the user has scrolled that pane since then.

Reply:
Good point! I've made a change that it only scrolls back to the original position if the user hasn't scrolled themselves during component mode, if they do scroll, it'll stay in the position that they scroll to instead.

Reply:
Oh, I was trying to suggest to _never_ change the scroll offset back to what it was, but I guess that works too.